server/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wise-old-man-server",
-  "version": "2.4.20",
+  "version": "2.4.21",
```

Comment:
no need to bump versions on the API since this doesn't really require a redeployment

Reply:
doesn't it? because i made a new public type and included it in client-js?